src/rgw/rgw_main.cc

```diff
@@ -1099,6 +1099,12 @@ int main(int argc, const char **argv)
   }
   r = rgw_perf_start(g_ceph_context);
 
+  r = percentile_perf_start(g_ceph_context);
```

Comment:
I'll call percentile_perf_start inside rgw_perf_start and percentile_perf_stop inside rgw_perf_stop.